ui/app/AppLayouts/Communities/views/CommunitiesGridView.qml

```diff
@@ -34,6 +35,11 @@ StatusScrollView {
         // values from the design
         readonly property int scrollViewTopMargin: 20
         readonly property int subtitlePixelSize: 17
+        readonly property int promotionalCardPosition: 2
```

Comment:
Not sure we should hardcode it like this, eventually the layout should be responsive with a flexible amount of columns. It should be reflected here imho (as e.g. `grid.columnCount - 1`)

Reply:
You are right, we can fix this way. I'll add a columnCount property.